lv_objx/lv_chart.c

```diff
@@ -45,6 +47,7 @@ static lv_signal_func_t ancestor_signal;
 /**********************
  *      MACROS
  **********************/
+#define strlens(s) (s==NULL?0:strlen(s))
```

Comment:
I would make this a more specific name like `lv_chart_strlen` to reduce the chances of conflict with someone else's function.